x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/upgrade_review_algorithms.md

```diff
@@ -52,6 +56,9 @@ Status: `in progress`.
 
 - **Merged version**: Also labeled as `merged_version`. This is the version of the rule that we determine via the various algorithms. It could contain a mix of all the rule versions on a per-field basis to create a singluar version of the rule containing all relevant updates and user changes to display to the user.
 
+- **Grouped fields**
+  - `data_source`: an object field that contains either a `data_view_id` string OR an `index` array of strings
```

Comment:
Now we probably want to update this line as well to mention the whole `data_source` object, same as in tables below.